glide.yaml

```diff
@@ -10,6 +10,10 @@ import:
   version: ~0.4
 - package: github.com/golang/mock
   version: master
+- package: github.com/grpc-ecosystem/grpc-opentracing
+  version: master
```

Comment:
grpc-opentracing isn't doing releases unfortunately :( we need to evaluate grpc-opentracing more anyways before grpc can get out of the x/ package